packages/cli/src/buildFlags.js

```diff
@@ -0,0 +1,26 @@
+const { flags } = require('@oclif/command');
+const { pickBy } = require('lodash');
+
+const baseFlags = {
+  format: flags.string({
+    char: 'f',
+    // TODO make these dynamic
+    options: ['plain', 'json', 'raw', 'row', 'table', 'small'],
+    default: 'table'
+  }),
+  debug: flags.boolean({
+    char: 'd'
+  })
+};
+
+// didn't destruture these opts because I want them all on one object to be picked from
+const defaultOpts = { debug: true, format: false };
+const buildFlags = (rawFlags = {}, opts = {}) => {
```

Comment:
This was the cleanest way I could think of to let each command opt into what global args they want to inherit. All of them want `--debug`, but only some care about `--format=x` (the ones with table output). It's not the most elegant, but it provides good defaults and is easy to tweak per-command (with minimal boilerplate).